.gitleaks.toml

```diff
@@ -196,6 +214,25 @@ files = [
   "test/util/ssl/(.*)", # test ssl certs
   "internal/plugin/connectors/tcp/mssql/connection_details_test.go", # fake cert string
 ]
+# As of v4, gitleaks can whitelist paths to accommodate no longer using
+# paths in the `files` whitelist.
+paths = [
+  "doc/full-demo/secrets",
+  "demos/full-demo/secrets",
+  "demos/quick-start/docker/etc",
+  "demos/k8s-demo/etc",
+  "test/pg_handler/etc",
+  "test/pg2_handler/etc",
+  "test/ssh_handler",
+  "test/ssh_agent_handler",
+  "test/connector/http/generic/certs",
+  "test/connector/ssh",
+  "test/connector/ssh_agent",
+  "test/connector/tcp/mssql/certs",
+  "test/ssh",
+  "test/util/ssl",
+  "internal/plugin/connectors/tcp/mssql"
+]
 regexes = [
```

Comment:
this section should be removed, right?

does the v4 gitleaks parse the commit exclusions too?

Reply:
We need to preserve any existing values for backwards compatibility until everyone has upgraded to v4.  v4 does parse the commit exclusions, I just translated existing file based exclusions rather than interpreting intention.